src/things/comment.rs

```diff
@@ -61,6 +61,19 @@ impl Shred for Comment {
             return;
         }
 
+        if config.prevent_comment_deletion {
+            debug!("Skipping DELETION due to `prevent_comment_deletion` filter ({})", config.prevent_comment_deletion);
+            match &config.gdpr_export_dir {
+                Some(_) => {}
+                None => {
+                    // #35 must be fixed prior to removing this disclaimer. But that has to do with Reddit's JSON functionality.
```

Comment:
Link to GitHub issue would be better here since others might no understand #35 is a reference to a GH issue.

Reply:
I figured that would be fine, since the only way they are getting this would be via github anyway, and that is common practice for how github links are referenced elsewhere. But changing it is non-issue.